server/routes/editTag.ts

```diff
@@ -4,37 +4,43 @@ import lookupCommunityIsVisibleToUser from '../util/lookupCommunityIsVisibleToUs
 import { factory, formatFilename } from '../util/logging';
 const log = factory.getLogger(formatFilename(__filename));
 
+export const Errors = {
+  NotLoggedIn: 'Not logged in.',
+  NoTagId: 'Must supply tag ID.',
+  NotAdmin: 'Must be an admin to feature tags.',
+  TagNotFound: 'Tag not found.'
+};
+
 const editTag = async (models, req: Request, res: Response, next: NextFunction) => {
   const [chain, community] = await lookupCommunityIsVisibleToUser(models, req.body, req.user, next);
   if (!req.user) {
```

Comment:
Since this route uses passport we don't need to check if the user object is on the request. It will be caught by passport if it isn't.

Reply:
Note here: https://662-251518273-gh.circle-artifacts.com/0/coverage/server/router.ts.html#L130

Reply:
Ah, thank ya!!